app/controllers/ProofOfAuthorityController.scala

```diff
@@ -72,10 +72,8 @@ class ProofOfAuthorityController @Inject() (
             (checkStateActor ? CheckState(request.internalId, LocalDateTime.now.plusSeconds(10), s)).mapTo[
```

Comment:
CLARIFICATION: 4 lines up - what does implicit val timeout = Timeout(10 seconds) do?

Reply:
It's a required implicit argument for the "Actor" magic.  There is another one within the `CheckStateActor` class (this time 30 secs).